pkg/prometheus/promcfg.go

```diff
@@ -2401,6 +2403,19 @@ func (cg *ConfigGenerator) appendScrapeProtocols(slice yaml.MapSlice) yaml.MapSl
 	return cg.WithMinimumVersion("2.49.0").AppendMapItem(slice, "scrape_protocols", cpf.ScrapeProtocols)
 }
 
+func (cg *ConfigGenerator) appendRuntime(slice yaml.MapSlice, runtime *monitoringv1.RuntimeConfig) yaml.MapSlice {
+	if runtime == nil || !cg.WithMinimumVersion("2.53.0").IsCompatible() {
+		return slice
+	}
+
+	var runtimeSlice yaml.MapSlice
+	if runtime.GoGC != nil {
+		runtimeSlice = cg.WithMinimumVersion("2.53.0").AppendMapItem(runtimeSlice, "gogc", *runtime.GoGC)
```

Comment:
(nit) the version is already checked before
```suggestion
		runtimeSlice = cg.AppendMapItem(runtimeSlice, "gogc", *runtime.GoGC)
```